.pre-commit-config.yaml

```diff
@@ -2,15 +2,15 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v5.0.0
+  rev: v6.0.0
   hooks:
   - id: trailing-whitespace
   - id: end-of-file-fixer
   - id: check-yaml
   - id: check-added-large-files
 
 - repo: https://github.com/markdownlint/markdownlint
-  rev: v0.11.0
+  rev: v0.12.0
```

Comment:
Let's not make this bump because of https://github.com/markdownlint/markdownlint/issues/521. E.g., on my machine the default ruby version is still 2.6.10, so bumping it would break local linting for me.